solidity/contracts/StaticOracle.sol

```diff
@@ -73,7 +73,7 @@ contract StaticOracle is IStaticOracle {
     uint32 period
   ) external override returns (address[] memory preparedPools) {
     preparedPools = _getPoolsForTiers(tokenA, tokenB, _knownFeeTiers);
-    _prepare(preparedPools, period);
+    _prepare(preparedPools, _getCadinalityForTimePeriod(period));
```

Comment:
I think it would be better if `prepare*WithTimePeriod` functions called their `prepare*WithCardinality` counterpart. It would reduce the amount of repeated code

Reply:
Oh. Nice catch Sam. Thanks. 🤦🏽‍♂️